src/status_im2/contexts/onboarding/profiles/style.cljs

```diff
@@ -0,0 +1,61 @@
+(ns status-im2.contexts.onboarding.profiles.style
+  (:require [quo2.foundations.colors :as colors]
+            [react-native.platform :as platform]
+            [quo2.foundations.shadows :as shadows]))
+
+;; Profiles Section
+
+(defn profiles-profile-card
+  [last-item?]
+  ;; This part needs to be improved, inverted shadow is not supported in android
+  ;; https://reactnative.dev/docs/shadow-props#shadowoffset-ios
+  (merge
+   (:shadow-1 (shadows/get-scales true :dark))
+   {:padding-horizontal 20
+    :margin-bottom      (when-not last-item? -24)}))
+
+(def profiles-container
+  {:position :absolute
+   :left     0
+   :top      0
+   :bottom   0
+   :right    0})
+
+(def profiles-header
+  {:flex-direction     :row
+   :padding-horizontal 20
+   :padding-top        112
+   :margin-bottom      20})
+
+(def profiles-header-text
+  {:color colors/white
+   :flex  1})
+
+;; Login Section
+
+(def login-container
+  {:position           :absolute
+   :left               0
+   :top                0
+   :right              0
+   :bottom             0
+   :padding-top        56
+   :padding-horizontal 20})
+
+(def multi-profile-button
+  {:align-self :flex-end})
+
+(def login-profile-card
+  {:margin-vertical 20})
+
+(def keyboard-avoiding-view
+  {:flex 1})
+
+(def info-message
+  {:margin-top 8})
+
+(defn login-button
+  []
+  {:margin-top    8
+   :margin-bottom (if platform/android? 20 46)})
```

Comment:
why difference?

Reply:
Position of action buttons in 46 in ios ([ref](https://www.figma.com/file/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=4325-306255&t=wiJrBPAUaBE4wQ6g-0)), but @pedro-et mentioned as android doesn't have bottom swipe bar, its bottom position should be same as left/right margin which is 20.
(Cases for android also using swipe bar, is still under [exploration](https://github.com/status-im/status-mobile/issues/15381))